packages/api/src/settings/entities.ts

```diff
@@ -40,7 +40,8 @@ export function getGenericSettingValueRealized(
       .split(',')
       .map((v: string) => Number.parseInt(v.trim()));
   } else if (this.type === 'Image') {
-    normalizedValue = `${process.env.SERVER_URL as string}/${this.value}`;
+    const apiURL = process.env.REACT_APP_API_URL;
+    normalizedValue = apiURL ? `${apiURL}/${this.value}` : `/${this.value}`;
```

Comment:
This conditional is unnecessary. We should just require the env variable to be defined via `env-required.ts`

Reply:
Leaving this setting empty is allowed when running in production - when running the api on the same domain as the frontend. We could leave out the conditional though and always require the full url to the api to be in that variable even in the case of running on the same domain. What do you think?

Reply:
Yeah I think always requiring the full url is preferable.

Reply:
Ok, done.